### ISSUE TITLE
reCAPTCHA

### ISSUE THREAD

**Issue by MaikeMota** [mtuanp/packtpubbot#6]
Since today morning, the packtpub is requesting to answer to reCAPTCHA to claim the free ebook.

Now we have to figure out how to bypass the verification or the bot only will be able to download the ebook after user claim manually 

Anyone has any suggestion or solution for this? 

![image](https://cloud.githubusercontent.com/assets/4526602/26462752/5fd2c936-4158-11e7-8fd6-d0ab67dc4c2d.png)

**Comment by PhillipMC** [comment]
Sorry I use your bot daily and I'm loving it, but my programming skills are very basics to help them although if I can help with something ask me.

**Comment by MaikeMota** [comment]
Hello again, I was studying some alternatives to bypass the Google Recaptcha V2 , but until now, all the possibilities finishes in a dead end. I'm wondering if it really has any chance to bypass or it's an impossible task for mortals.

For now, I think the best we can do is rewrite the bot code to became an notifier about the book of the day, and then the user can see the title and decide about access the Packtpub page and claim it manually.

We also can create a Telegram's group and allow the bot send to all members the book of the day with the respectives links.

**Comment by SpEcHiDe** [comment]
You can get your daily dose of free e-books here: https://telegram.dog/PIT_PacktPubFreeLearning

**Comment by MaikeMota** [comment]
@SpEcHiDe Yes, you are right. But the bot was capable of claim and downloading the ebook and store where you want, without any human interference. Most people like me forget of claim the ebook.
